docs/getting-started.rst

```diff
@@ -1,56 +1,94 @@
-Getting Started
-===============
 
-Demo
-----
 
-If you'd like to check out a demo of DefectDojo before installing it, you can check out our `PythonAnywhere demo site`_.
 
-.. _PythonAnywhere demo site: https://defectdojo.pythonanywhere.com
+Change into the newly created ```django-DefectDojo``` directory:
 
-You can log in as an administrator like so:
+    ``cd django-DefectDojo/``
 
-.. image:: /_static/admin-creds.png
+There is a script in the main folder called ``setup.bash`` that will allow you to interactively install DefectDojo on any Linux based systems. We do not recommend running DefectDojo as root, but you may do so if you choose. 
 
-You can also log in as a product owner / non-staff user:
+**You will need:**
+* MySQL
+* pip
 
-.. image:: /_static/prod-owner-creds.png
+**Recommended**
+* virtualenv
 
-Installation
-------------
+1. If you haven't already, run ``mysql_secure_install`` to set a password for your root MySQL user.
 
-.. _debian-or-rhel-based-bash-install-script:
+2. Create a MySQL user with CREATE priviledges, or use root.
 
-Debian or RHEL based Bash Install Script
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**Run the ``setup.bash`` script**
+This script will:
 
-There is a script in the main folder called `setup.bash` that will allow you to
-interactively install DefectDojo on many Linux-based systems. We do not
-recommend running DefectDojo as root, but you may do so if you choose. This is
-the quick version of the installation instructions, but if you want more details
-about what's going on, check out `this wiki page`_ on Ubuntu 14.04 installation
-(most steps should be applicable to other distributions as well).
+1. Install all the operating system packages needed
 
-.. _this wiki page: https://github.com/rackerlabs/django-DefectDojo/wiki/DefectDojo-Installation-Guide---Ubuntu-Desktop-14.04
+2. Prompt for database connection information and create the necessary table
 
```

Comment:
The wiki info is not needed. All other info is merged to the ubuntu/linux specific instructions.